### PR TITLE
TimeMap iterate only over SCHEDULE section

### PR DESCRIPTION
**Pull request by joakim-hove**
Small improvement

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Schedule/TimeMap.cpp [hunk 1]
```diff
@@ -115,7 +116,8 @@ namespace {
             }
         }
 
-        for( const auto& keyword : deck ) {
+        SCHEDULESection schedule(deck);
+        for( const auto& keyword : schedule ) {
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Schedule/TimeMap.cpp]: I suppose you could write this even more succinctly as
```C++
for (const auto& keyword : SCHEDULESection(deck)) {
    // ...
}
```

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2307 please

File: src/opm/parser/eclipse/EclipseState/Schedule/TimeMap.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/TimeMap.cpp]: Done

**Pull request merged**